tests/bats/Makefile

```diff
@@ -0,0 +1,71 @@
+#  SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#  SPDX-License-Identifier: Apache-2.0
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+include $(CURDIR)/versions.mk
+include $(CURDIR)/common.mk
+
+BATS_IMAGE = batstests:$(GIT_COMMIT_SHORT)
+
+TEST_CHART_REPO ?= "oci://ghcr.io/nvidia/k8s-dra-driver-gpu"
+TEST_CHART_VERSION ?= $(VERSION_GHCR_CHART)
+TEST_CHART_LASTSTABLE_REPO ?= "oci://ghcr.io/nvidia/k8s-dra-driver-gpu"
+TEST_CHART_LASTSTABLE_VERSION ?= "25.3.2-2c250af3-chart"
```

Comment:
Do we really want this hard-coded?

Reply:
In this first version of the patch that I would have liked to land before today: yes. For now. For now means that this easily changes tomorrow or next week.

I want to be able to iterate on this fast, in a controlled fashion.

Reply:
on topic: my plan is to not have this hard-coded eventually, but to have a heuristic. I don't know yet if this is always the actual last stable, or the HEAD of the last-stable branch, or an inbetween (in which case manual bumps might still be worth considering).